libs/shared/components/src/Markdown/markdownOptions.tsx

```diff
@@ -11,6 +11,7 @@ import * as styles from './Markdown.css'
 
 const markdownOverrides: MarkdownToJSX.Overrides = {
   ul: BulletList,
+  ol: BulletList,
```

Comment:
This needs to render as `<BulletList type="ol">`, right?

Reply:
ofcoooourse :D fix coming up!